packages/algob/test/builtin-tasks/run.ts

```diff
@@ -31,30 +31,62 @@ describe("Run task", function () {
 
 	it("Should run the scripts to completion", async function () {
 		await this.env.run(TASK_RUN, {
-			script: "./scripts/async-script.js",
+			script: ["./scripts/async-script.js"],
 		});
 	});
 
 	it("Should run the script without arguments", async function () {
 		await this.env.run(TASK_RUN, {
-			script: "./scripts/async-script.js",
+			script: ["./scripts/async-script.js"],
 		});
 	});
 
 	it("Should run the script with empty arguments", async function () {
 		await this.env.run(TASK_RUN, {
-			script: "./scripts/async-script.js",
-			args: []
+			script: ["./scripts/async-script.js"],
+			arg: ""
 		});
 	});
 
 	it("Should run the script with arguments", async function () {
 		await this.env.run(TASK_RUN, {
-			script: "./scripts/async-script.js",
-			args: ["arg1", "arg2"]
+			script: ["./scripts/async-script.js"],
+			arg: '{"name":"user"}'
 		});
 	});
 
+	it("Should fail if a script is not passed as first element of script array", async function () {
+		await expectBuilderErrorAsync(
+			async () =>
+				await this.env.run(TASK_RUN, {
+					script: ["arg1", "./scripts/does-not-exist"],
+				}),
+			ERRORS.BUILTIN_TASKS.RUN_FILES_NOT_FOUND,
+			"arg1"
+		);
+	});
+
+	it("Should fail if empty script array is passed", async function () {
+		await expectBuilderErrorAsync(
+			async () =>
+				await this.env.run(TASK_RUN, {
+					script: [],
+				}),
+			ERRORS.BUILTIN_TASKS.RUN_FILE_NOT_FOUND_WITH_SUGGESTION,
+		);
+	});
+
+	it("Should throw error when a valid JSON string is not passed as argument in the script", async function () {
+		await expectBuilderErrorAsync(
+			async () =>
+				await this.env.run(TASK_RUN, {
+					script: ["./scripts/async-script.js"],
+					arg: "{name: usersname}"
```

Comment:
why is it invalid json?

Reply:
It should be like `{"name": "username"}`